backend/main.go

```diff
@@ -27,16 +41,30 @@ func getFellowHandler(w http.ResponseWriter, req *http.Request) {
 	authorized := isAuthorized(w, req)
 
 	if !authorized {
```

Comment:
Maybe `if !isAuthorized(w, req) {` instead?
```suggestion
	if !isAuthorized(w, req) {
```

Reply:
Sure I'll include that